README.md

```diff
@@ -167,4 +169,16 @@ using your real name (sorry, no pseudonyms or anonymous contributions.)
 
 You can add the sign off when creating the git commit via `git commit -s`.
 
+### Commit Style
+
+Simple house-keeping for clean git history.
+
+1. Separate subject from body with a blank line
+2. Limit the subject line to 50 characters
```

Comment:
On Wed, Sep 09, 2015 at 04:41:54PM -0700, Vincent Batts wrote:
> > +2. Limit the subject line to 50 characters
> 
> Regardless of the past commits, this would obviously be for future
> commits.

This limit has been part of the Git docs for a while now, and they're
still not keeping under 50 (they also reference the 50-char soft limit
in Documentation/SubmittingPatches).  I'd prefer just putting a
hard-limit at 70 chars in your validation tool (#167), but I'm ok
putting a limit of 50 in that validation tool (and still merging if
"commit summary is too long" is the only error).  I just think wedging
useful summaries into 50-len(prefix) is going to be more trouble than
its worth.